src/components/Layout/Header/ActionCenter/components/ActionStatusTag.tsx

```diff
@@ -59,10 +59,16 @@ export const ActionStatusTag = ({ status }: ActionStatusTagProps) => {
         )
       case ActionStatus.ClaimAvailable:
         return (
-          <Tag size='sm' colorScheme='blue' {...defaultTagProps}>
+          <Tag size='sm' colorScheme='green' {...defaultTagProps}>
```

Comment:
thinking: Make sense even thought it's not the end of the claim but it can be considered as a positive state